js/msp/MSPHelper.js

```diff
@@ -1827,6 +1681,51 @@ MspHelper.prototype.sendRxFailConfig = function(onCompleteCallback) {
         MSP.send_message(MSPCodes.MSP_SET_RXFAIL_CONFIG, buffer, false, nextFunction);
     }
 }
+DataView.prototype.offset = 0;
```

Comment:
I think these should better go into `js/injected_methods.js`. Also, I think the distinction between `readXX` and `readSXX` is not very clear. It seems to be the more common case to assume the default is signed, and unsigned is a special case (cf. `int` and `uint`), so I'd rather go with `readUXX` (unsigned) and `readXX` (signed), as I think people will more easily understand that without looking up the definition. Your thoughts?